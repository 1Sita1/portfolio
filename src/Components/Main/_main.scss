@use '../../Sass/Abstracts/mixins' as *;

.main-section {
    @include flex(center, center);
    flex-wrap: wrap;
    background-color: #0a192f;
    height: 90vh;
}

.main-section > article {
    margin: 0 60px;
}

.bitmoji-container > img {
    width: 400px;
    max-width: 80vw;
    height: auto;
}