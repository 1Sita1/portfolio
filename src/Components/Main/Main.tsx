import { type } from "@testing-library/user-event/dist/type";
import { FunctionComponent } from "react";
import './_main.scss'

type MainProps = {

}

const Main: React.FC = () => {

    return (
        <section className="fullscreen main-section">
            <article>
                <h1>Hi</h1>
                <h2>I am Arsenii</h2>
                <h2>And I love Web</h2>
            </article>
            <article>
                <figure className="bitmoji-container">
                    <img src="/Img/bitmoji.png"></img>
                </figure>
            </article>
        </section>
    )
}

export default Main