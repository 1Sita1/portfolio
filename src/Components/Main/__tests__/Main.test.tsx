import { render, screen } from '@testing-library/react'
import TestRenderer from 'react-test-renderer'
import Main from "../Main"

describe("Main tests", () => {
    it("renders correctly", () => {
        const component = TestRenderer.create(
            <Main />
        )
        const tree = component.toJSON()
        expect(tree).toMatchSnapshot()
    })
})