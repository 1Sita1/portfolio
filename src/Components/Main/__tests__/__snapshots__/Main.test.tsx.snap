// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main tests renders correctly 1`] = `
<section
  className="fullscreen main-section"
>
  <article>
    <h1>
      Hi
    </h1>
    <h2>
      I am Arsenii
    </h2>
    <h2>
      And I love Web
    </h2>
  </article>
  <article>
    <div
      style={
        Object {
          "backgroundColor": "brown",
          "height": "400px",
          "width": "400px",
        }
      }
    />
  </article>
</section>
`;
